discord/guild.py

```diff
@@ -3816,3 +3816,32 @@ async def create_auto_moderation_rule(
             self.id, payload, reason=reason
         )
         return AutoModRule(state=self._state, data=data)
+
+    async def delete_auto_moderation_rule(
+        self,
+        *,
+        id: int,
```

Comment:
`id` should be positional
```suggestion
        id: int,
        *,
```